perses/app/relative_point_mutation_setup.py

```diff
@@ -362,7 +362,10 @@ def __init__(self,
                                                         validate_energy_bookkeeping=validate_bool)
 
             # Check for charge change...
-            self._handle_charge_changes(topology_proposal, new_positions)
+            if phase != 'vacuum':
```

Comment:
Can you add a sanity check to this method that checks to make sure that `phase` is valid?
```python
KNOWN_PHASES = ['vacuum', 'complex', 'solvent']
if not phase in KNOWN_PHASES:
    raise ValueError(f"phase '{phase}' unknown; must be one of {KNOWN_PHASES}")
```

Reply:
Done!